napari/util/io.py

```diff
@@ -32,6 +36,10 @@ def magic_imread(filenames, *, use_dask=None, stack=True):
     image : array-like
         Array or list of images
     """
+    # cast Path to string
+    if isinstance(filenames, Path):
+        filenames = filenames.as_posix()
```

Comment:
Q: does `as_posix()` work on Windows?

Reply:
Never mind, I just realised that your tests and our Windows CI answered yes to this question. =D